generator.go

```diff
@@ -288,7 +288,7 @@ func (g *generator) genEnum(name string, v cue.Value) []ts.Decl {
 	// We restrict the expression of TS enums to CUE disjunctions (sum types) of strings.
 	allowed := cue.StringKind | cue.NumberKind | cue.NumberKind
```

Comment:
this is weird. while we're in here, could we fix this? seems like this should be:
```suggestion
	allowed := cue.StringKind | cue.IntKind
```
though i am cognizant that the kinds of numeric values are less than dependable, see e.g. grafana/thema#111